### PR TITLE
remove default account from side menu bar

### PR DESCRIPTION
**Pull request by SehKeat**
Remove default account from side menu bar.
Other way to change default account is through navigate account from navigate side bar.

### CODE REVIEW

**Review by anakin001 — COMMENTED** [review]
Please remove unnecessary lines of codes(e.g. the popup modal from the default acc)

**Comment by SehKeat** [comment]
noted

File: src/components/headerComponent.vue [hunk 1]
```diff
@@ -228,16 +228,6 @@ export default defineComponent({
...
-    emitter.on('TRIGGER_SWITCH_DEFAULT_ACCOUNT_MODAL', (payload) => {
-      openSetDefaultModal.value = payload;
-    });
-
-    emitter.on('CLOSE_SET_DEFAULT_ACCOUNT_MODAL', payload => {
-      if(payload){
-        openSetDefaultModal.value = false;
-      }
-    });
-
```
Comment (anakin001) [inline src/components/headerComponent.vue:240]: is SetAccountDefaultModal.vue still a relevant? if not, please remove it

Comment (anakin001) [reply src/components/headerComponent.vue:240]: also remove the variable 'openSetDefaultModal'

Comment (SehKeat) [reply src/components/headerComponent.vue:240]: noted, just revised and remove SetAccountDefaultModal.vue

**Review by anakin001 — APPROVED** [review]

**Pull request merged**